qos-monitor/src/main/java/eu/arrowhead/core/qos/dto/event/monitoringevents/FinishedMonitoringMeasurementEventDTO.java

```diff
@@ -29,4 +32,14 @@ public FinishedMonitoringMeasurementEventDTO() {
 	//-------------------------------------------------------------------------------------------------
 	public void setPayload(final List<IcmpPingResponseDTO> payload) { this.payload = payload; }
 
+	//-------------------------------------------------------------------------------------------------
+	@Override
+	public String toString() {
+		return new StringJoiner(", ", InterruptedMonitoringMeasurementEventDTO.class.getSimpleName() + "[", "]")
```

Comment:
InterruptedMonitoringMeasurementEventDTO.class.getSimpleName() is wrong here

Reply:
done